app/models/picture_set.rb

```diff
@@ -38,6 +40,10 @@ def create
       until !jobs.any?(&:status) do end
       # Merge all polaroid previews to an animated gif
       Syscall.execute("time convert -delay 60 #{date}_*#{POLAROID_SUFFIX} #{date}#{ANIMATION_SUFFIX}", dir: dir)
+      # Merge all images in one combined image
+      Syscall.execute("montage -geometry '25%x25%+25+25<' -background 'LightCoral' " \
+      "-title '#{OPTS.image_caption}' -font '#{OPTS.image_font}' -fill 'white' -pointsize 72 -gravity 'Center' " \
+      "#{date}_*.jpg #{date}#{COMBINED_SUFFIX}")
```

Comment:
I suggest to call it like this: 
```
      # Merge all images in one combined image
      Thread.new do
        Syscall.execute("time montage -geometry '25%x25%+25+25<' -background 'LightCoral' " \
          "-title '#{OPTS.image_caption}' -font '#{OPTS.image_font}' -fill 'white' -pointsize 72 -gravity 'Center' " \
          "#{date}_*.jpg #{date}#{COMBINED_SUFFIX}", dir: dir)
      end
``` 
to offload the work to a background thread (montage takes 14s on my raspi). 
Added correct working dir.